.github/dependabot.yml

```
@@ -0,0 +1,22 @@
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "07:00"
+      timezone: "UTC"
+    labels: ["dependencies", "github-actions"]
+    open-pull-requests-limit: 5
+    rebase-strategy: "auto"
+    directory: "/" # Location of package manifests
```

YAMLlint (1.37.1)

[error] duplication of key "directory" in mapping

(key-duplicates)